content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -400,18 +399,90 @@ $ curl \
 }
 ```
 
+## Show workspace
+
+This endpoint shows details for a workspace in the organization.
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /organizations/:organization/workspaces/:name |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - Specifies the organization name under which to list the workspaces. This is specified in the URL path.
+- `:name` (`string: <required>`) - Specifies the name of the workspace to show details for. This is specified in the URL path.
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/organizations/my-organization/workspaces/my-workspace-3
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "ws-mD5bmJ8ry3uTzuHi",
+    "type": "workspaces",
+    "attributes": {
+      "name": "my-workspace-3",
+      "environment": "default",
+      "auto-apply": false,
+      "locked": false,
+      "created-at": "2018-03-08T22:30:00.404Z",
+      "working-directory": null,
+      "terraform-version": "0.11.3",
+      "permissions": {
+        "can-update": true,
+        "can-destroy": true,
+        "can-queue-destroy": true,
+        "can-queue-run": true,
+        "can-update-variable": true,
+        "can-lock": true,
+        "can-read-settings": true
+      },
+      "actions": {
+        "is-destroyable": true
+      }
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "my-organization",
+          "type": "organizations"
+        }
+      },
+      "latest-run": {
+        "data": null
+      },
+      "current-run": {
+        "data": null
+      }
+    },
+    "links": {
+      "self": "/api/v2/organizations/my-organization/workspaces/my-workspace-3"
+    }
+  }
+}
+
+```
+
 
 ## Delete a workspace
 
 This endpoint deletes a workspace.
 
 | Method | Path           |
 | :----- | :------------- |
-| DELETE | /organizations/:organization/workspaces/:workspace |
+| DELETE | /organizations/:organization/workspaces/:name |
 
 ### Parameters
 
-- `:workspace` (`string: <required>`) - Specifies the workspace ID to delete. This parameter is specified in the URL path.
+- `:name` (`string: <required>`) - Specifies the name of the workspace to delete. This parameter is specified in the URL path.
 - `:organization` (`string: <required>`) - Specifies the organization name where the workspace belongs. This parameter is specified in the URL path.
```

Comment:
I'd probably say "the name of the organization the workspace belongs to", but since this isn't actually changing, either way is fine.

Reply:
Agree on this and will do new commit with the change.